nx.json

```diff
@@ -18,17 +18,17 @@
 	},
 	"tasksRunnerOptions": {
 		"default": {
+			"runner": "nx-cloud",
 			"options": {
-				"accessToken": "OTdkYmFjMTYtNGFhNS00MjM2LTg0OGMtZmNhNGZiMmU2YTM5fHJlYWQtd3JpdGU=",
 				"cacheableOperations": [
 					"build",
 					"lint",
 					"test",
 					"e2e",
 					"build-storybook"
-				]
-			},
-			"runner": "nx-cloud"
+				],
+				"accessToken": "ZDI5NGQyNWMtYzRiNS00YjM0LWJkNDItMWEzMGQ2NWMwNWJkfHJlYWQtb25seQ=="
```

Comment:
This token is read-only as recommended by NX: https://nx.dev/nx-cloud/account/scenarios#security-scenarios